s4e-web/cypress/integration/settings/settings-institution-profile.spec.ts

```diff
@@ -1,91 +1,138 @@
 /// <reference types="Cypress" />
 
 import { Login } from '../../page-objects/auth/auth-login.po';
-import { InstitutionSearch } from '../../page-objects/settings/settings-institution-search.po';
-import { SideNav } from '../../page-objects/settings/settings-side-nav.po';
-import { InstitutionProfile } from '../../page-objects/settings/settings-institution-profile.po';
-import promisify from 'cypress-promise';
-import { InstitutionFactory } from '../../../src/app/views/settings/state/institution/institution.factory.spec';
+import { UserOptionsGoToSettings } from "../../page-objects/user-options/user-options-go-to-settings-profile.po";
+import { SettingsInstitutions } from "../../page-objects/settings/settings-institution-profile.po";
+import { SettingsNav } from "../../page-objects/settings/settings-navigation.po";
+import { ConfirmModal } from "../../page-objects/modal/confirm-modal.po"
+import { SettingsAdministratorPrivilage } from "../../page-objects/settings/settings-superadmin-add-privilege.po"
+import { UserOptionsAuthentication } from '../../page-objects/user-options/user-option-authentication.po';
 
-// context.skip('Settings institution profile', () => {
-//   beforeEach(() => {
-//     cy.fixture('users/zkAdmin.json').as('zkAdmin');
-//   });
+before(() => {
+  cy.fixture('users/zkAdmin.json').as('zkAdmin');
+  cy.fixture('users/admin.json').as('admin');
+  cy.fixture('institution.json').as('institution');
```

Comment:
Shouldn't this fixture have plural name? I.e. `institutions` instead of a singluar `institution`.